tests/unit/v1/test_base_query.py

```diff
@@ -864,9 +861,10 @@ def test__normalize_cursor_w___name___wo_slash(self):
 
     def test__to_protobuf_all_fields(self):
         from google.protobuf import wrappers_pb2
-        from google.cloud.firestore_v1.gapic import enums
-        from google.cloud.firestore_v1.proto import document_pb2
-        from google.cloud.firestore_v1.proto import query_pb2
+        from google.cloud.firestore_v1.types import StructuredQuery
+
+        from google.cloud.firestore_v1.types import document
+        from google.cloud.firestore_v1.types import query
```

Comment:
It looks to me as though we use `StructuredQuery` and `query.StructuredQuery` interchangeably. Are these functionally equivalent?

Reply:
Yep.